charts/spin-operator/templates/NOTES.txt

```diff
@@ -13,11 +13,11 @@ already done so, please ensure the following:
 
 1. Install the containerd-shim-spin SpinAppExecutor:
 
-  $ kubectl apply -f https://raw.githubusercontent.com/spinkube/spin-operator/main/config/samples/spin-shim-executor.yaml
+  $ kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.1.0/spin-operator.shim-executor.yaml
```

Comment:
we should be able to use a variable here instead of hardcoded release number.

Reply:
Great point @rajatjindal 

I see that the chart version does not include the `v` and therefore the update leaves the `v` in place and adds the chart version after it.

![Screenshot 2024-04-11 at 13 04 12](https://github.com/spinkube/spin-operator/assets/9831342/ee7f3bab-3ce9-485b-a99d-9e9363d57950)


<img width="626" alt="Screenshot 2024-04-11 at 13 02 37" src="https://github.com/spinkube/spin-operator/assets/9831342/0df86d38-4a78-4b90-8ba0-0ca7a5da0bc0">